runtime/structure.c

```diff
@@ -90,3 +35,32 @@ static bool tidListContains(const Tid *start, const Tid *end, Tid tid) {
     // start == end, so there is no such member
     return false;
 }
+
+TaggedPtr structCreateFiller(FillerDescPtr fdp, Fillability *fillability) {
```

Comment:
All external symbols must start with `_bal_`

Reply:
fixed (changed to internal symbol)